app/models/user.rb

```diff
@@ -136,4 +176,12 @@ def object_attrs_for_paper_trail(attributes)
   def set_token
     self.token = SecureRandom.hex
   end
+
+  private_class_method def self.permissions_count(project_id, inherited, users, permissions_projects = [])
```

Comment:
this look copy-pasty ... seems to duplicate logic from above ... can we reuse that somehow ?